Automattic/jetpack
# Search package: changes to the duplicated components (3/3)

#### Changes proposed in this Pull Request:
- Removed unnecessary dependencies from copied components.
- Added simple tests for components
- Make form toggle component grid layout compatible

#### Jetpack product discussion
n/a

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Run `(cd projects/packages/search && pnpm test)`
* Ensure all tests pass
* Will deal with version conflicts and change log when merging to master
